src/debug/PythonDebugProvider.ts

```diff
@@ -39,7 +39,7 @@ async function getPythonCommand(host: string, port: number): Promise<string> {
 
         // tslint:disable-next-line:strict-boolean-expressions
         if (pyExtension.exports && pyExtension.exports.debug) {
-            return (await pyExtension.exports.debug.getRemoteLauncherCommand(host, port)).join(' ');
+            return (await pyExtension.exports.debug.getRemoteLauncherCommand(host, port, false)).join(' ');
         } else {
```

Comment:
I'm about ready to approve, but one more question for ya. I can only assume the Python extension defaults this to "true" for a reason, so what are the downsides or risks of setting it to "false" and why don't those apply to us?

Reply:
If a customer breakpoints the **import statement** from very top of their code, the debugger may not be able to stop there since the debugger may attach after a while.

But to prevent any unforeseen issue, I introduce a feature flag **azureFunctions.waitForPythonDebugger**. This is set to false by default, but the customer can always set it to true to revert the new behavior.

Reply:
It would more likely be a problem if they set a breakpoint at the top of the _worker_ code, right? Not _their_ code? And I don't see any reason they would try to debug the worker code. The only reason debugging will work with the "--wait-for-client" flag is because of the "host lock lease" message and I want us to move away from that in favor of "worker started and initialized". I appreciate the attempt to be more careful and add the setting, but that's not really what I was going for. Sounds like the worst case is the debugger will still attach and the vast majority of breakpoints will still work, except for _maybe_ one at the beginning. With that in mind, I'd rather not add a setting that relies on the "host lock lease" message

Reply:
Yeah, sorry for the confusion. If the customer debug their code, they will not run into this issue.